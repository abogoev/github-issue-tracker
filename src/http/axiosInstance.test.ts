import axiosInstance from "./axiosInstance";

describe("axiosInstance", () => {
  it("should match snapshot", () => {
    expect(axiosInstance.defaults).toMatchSnapshot();
  });
});
