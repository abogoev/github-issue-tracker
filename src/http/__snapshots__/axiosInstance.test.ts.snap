// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`axiosInstance should match snapshot 1`] = `
Object {
  "adapter": [Function],
  "baseURL": "https://api.github.com",
  "headers": Object {
    "common": Object {
      "Accept": "application/json, text/plain, */*",
    },
    "delete": Object {},
    "get": Object {},
    "head": Object {},
    "patch": Object {
      "Content-Type": "application/x-www-form-urlencoded",
    },
    "post": Object {
      "Content-Type": "application/x-www-form-urlencoded",
    },
    "put": Object {
      "Content-Type": "application/x-www-form-urlencoded",
    },
  },
  "maxBodyLength": -1,
  "maxContentLength": -1,
  "timeout": 5000,
  "transformRequest": Array [
    [Function],
  ],
  "transformResponse": Array [
    [Function],
  ],
  "transitional": Object {
    "clarifyTimeoutError": false,
    "forcedJSONParsing": true,
    "silentJSONParsing": true,
  },
  "validateStatus": [Function],
  "xsrfCookieName": "XSRF-TOKEN",
  "xsrfHeaderName": "X-XSRF-TOKEN",
}
`;
